src/MHTimer.js

```diff
@@ -496,11 +496,17 @@ function parseUserMessage(message) {
 
         // Display or update the user's reminders.
         case 'remind':
+            var dataCatcher;
```

Comment:
use `let` instead of `var` -- you should then also wrap the case block in braces